tests/api_connexion/endpoints/test_log_endpoint.py

```diff
@@ -81,6 +81,18 @@ def setup_attrs(self, configured_app, configure_loggers, dag_maker, session) ->
 
         configured_app.dag_bag.bag_dag(dag, root_dag=dag)
 
+        with dag_maker(
+            f'{self.DAG_ID}_copy', start_date=timezone.parse(self.default_time), session=session
+        ) as dummy_dag:
+            DummyOperator(task_id=self.TASK_ID)
```

Comment:
What is this for?

Reply:
@uranusjr This is for adding new task which has different `dag_id` and same `task_id`.

Reply:
Does I need to create new testcase class for checking "Log endpoint API raise exception when same task_id with different dag_id."?

Reply:
So the DAG is initialised so that we can be sure tests in the class can correctly pick up the correct logs? That makes sense to me, but it’d be best to add a comment describing why this is needed. Otherwise someone might ends up accidentally removing this as “unneeded code” in the future.

Reply:
@uranusjr OK! I will add some comments at that code.